modules/v2.13.0/azure-mysql/data.tf

```diff
@@ -0,0 +1,13 @@
+data "azurerm_client_config" "current" {}
+
+# My public IP: data.http.myip.response_body
+data "http" "myip" {
+  url = "https://ipecho.net/plain"
```

Comment:
What is this for?

Reply:
get your current ip so that you can interact with the database server